root/app/cloudflare.sh

```diff
@@ -15,7 +15,7 @@ getPublicIpAddress() {
 
   # if dns method fails, use http method
   if [ "$IP_ADDRESS" = "" ]; then
-    IP_ADDRESS=$(curl -sf4 https://diagnostic.opendns.com/myip)
+    IP_ADDRESS=$( curl -sf ipinfo.io | jq -r '.ip')
```

Comment:
The `-4` ensures that the call is made over IPv4. Currently ipinfo.io does not have any AAAA records, so it still works, but it's probably best to keep the `-4` flag just incase the owner of ipinfo decides to enable IPv6 later on.

Reply:
Happy to reinstate 4, though this curl switch is the resolution of the ipinfo.io address rather than a prescriptive instruction to the service to only yield ipv4 or ipv6 host addresses in the json response.